DataFormats/common/include/CommonDataFormat/InteractionRecord.h

```diff
@@ -95,16 +103,122 @@ struct InteractionRecord {
     return (int64_t(orbit) * o2::constants::lhc::LHCMaxBunches) + bc;
   }
 
-  bool operator>(const InteractionRecord& other) const
+  bool operator>(const InteractionRecord other) const
   {
     return (orbit == other.orbit) ? (bc > other.bc) : (orbit > other.orbit);
   }
 
-  bool operator<(const InteractionRecord& other) const
+  bool operator>=(const InteractionRecord other) const
```

Comment:
is there any particular reason not to use `const&` anymore?

Reply:
In theory, since the size of this class is the same as of its reference, passing by value should be faster since the in both cases 8B are passed but passing the reference needs also dereferencing. In practice (just checked) both ways take exactly the same time (since the methods are inlined and probably the compiler would take care of optimal solution anyway).
So, reverted to passing the reference (+ fixed a bug).